src/pudl/analysis/plant_parts_eia.py

```diff
@@ -463,6 +468,8 @@ def get_gens_mega_table(self, mcoe):
             validate="m:1",
             how="left",
         )
+        all_gens.loc[:, "operating_year"] = all_gens["operating_date"].dt.year
+        all_gens = all_gens.astype({"operating_year": "Int64"})
```

Comment:
I'm not sure this is the best place in `MakeMegaGenTbl` to create the `operating_year` column but this is where I've put it for now. And I made it an `Int64`.

Reply:
I think here makes the most sense!